business/istio_validations.go

```diff
@@ -2,13 +2,15 @@ package business
 
 import (
 	"fmt"
+	"github.com/kiali/kiali/config"
```

Comment:
FYI: this doesn't follow our standard practice of putting the kiali imports in the bottom section. Same with the tls.go imports.

Reply:
Isn't it something that can be handled automatically for us in goland (i think it's what lucas is using) ? Personally with vscode I almost never have this issue as imports are automatically grouped as we want on save.

Reply:
I had to install a workspace in the IDE and I had an "automatically optimization" in the imports.
Thanks for the pointer.